features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/ItemDetails.kt

```diff
@@ -208,7 +208,7 @@ class ItemDetails(context: Context, attrs: AttributeSet): LinearLayout(context,
                 payBtn.text = context.getText(R.string.explore_pay_with_dash)
                 payBtn.setOnClickListener { onSendDashClicked?.invoke(true) }
             } else {
-                payBtn.isVisible = !merchant.deeplink.isNullOrBlank()
+                payBtn.isVisible = true//!merchant.deeplink.isNullOrBlank()
```

Comment:
in the new explore db, the `deeplink` field is empty, so we shouldn't use it to hide/show the Pay/Purchase button.

Reply:
There may be another field that should be checked instead